src/streaming/controllers/StreamController.js

```diff
@@ -582,7 +582,7 @@ function StreamController() {
 
             if (mediaInfo) {
                 voAdaptation = adapter.getDataForMedia(mediaInfo);
-                useCalculatedLiveEdgeTime = dashManifestModel.getRepresentationsForAdaptation(voAdaptation)[0].useCalculatedLiveEdgeTime;
+                useCalculatedLiveEdgeTime = dashManifestModel.getRepresentationsForAdaptation(voAdaptation, 1)[0].useCalculatedLiveEdgeTime;
```

Comment:
Given the requirement for setting the max number of representation requirements only comes from this line, I think could makes more sense adding a method to dashManifestModel that returns useCalculatedLiveEdgeTime of a given Adaptation and abstract StreamController of the way it should be calculated (useCalculatedLiveEdgeTime of the first representation of the adaptation). @Gontran-Molotov, what do you think?